File: UrbanSync.Server/DTO/ReportIssueDto.cs
using UrbanSync.Server.Models;

namespace UrbanSync.Server.DTO {

    public record ReportIssueDto(
        string Location,
         string Description,
          string? Image,
          string Status,
          string MunicipalityLevel,
           string MunicipalitySector,
            DateTime DateOfCreation  ) {

    }
}

File: urbansync.client/src/models/reportIssueDto.js
export default class reportIssueDto {
  constructor(
    location,
    description,
    image,
    status,
    municipalityLevel,
    municipalitySector,
    
  ) {
    this.Location = location, 
      this.Description = description,
      this.Image = image, 
      this.Status = status,
      this.MunicipalityLevel = municipalityLevel,
      this.MunicipalitySector = municipalitySector
  }
}
